rfcs/text/0000-rfc-process-updates.md

```diff
@@ -0,0 +1,109 @@
+
+Additionally, with the encouraged behavior of submitting draft pull requests prior to RFC readiness for review, there is no way to clearly designate the fact that the RFC should not be reviewed at all until the PR is no longer a draft.
+
+We propose adding the following new labels: 
+
+**rfc-draft** : This indicates that the authors are still actively working on the RFC, that the RFC is in a draft state, and readers should **not** comment on the RFC yet. Additionally this state serves to indicate more clearly the backlog of RFCs in development to avoid duplication of effort of overlapping RFCs and to help elicit offers of co-authorship.
+ 
+Currently "pre-RFCs" are often worked on in private Google Docs. To indicate more clearly the to the community the backlog of RFCs in development, avoid duplication of effort of overlapping RFCs and to help encourage co-authorship and early collaboration, draft RFCs should post a title and problem statement or motivation. They need not be full fledged RFCs with every section populated.
+
+**rfc-deprecated** : This indicates that the design is no longer implemented or executed in the DCP and has been usurped by a newer design or process. Note that the deprecated label is not standalone; it may be used in conjunction with  "rfc-approval" label for example.
+
+**rfc-paused** : This indicates that an RFC has been paused for further refinement. This roughly implies that an RFC has gone from being in a review status to a "pre-rfc" status and should not be reviewed further until designated so.
+
+**rfc-withdrawn** : This label is applied when an RFC has not finished the review process (and therefore has not been  explicitly rejected) but has been voluntarily withdrawn.
+
+Like the existing RFC process, any change of state to the RFC must be announced on the DCP slack channel. Also note that labeling an RFC has no implications on whether the PR is open or closed. Generally, RFCs in a terminal state (i.e. "approved", "declined", or "withdrawn").
+
+### 2. Reduce the RFC minimum review time
+
+The current RFC process requires a minimum of 2 weeks in DCP Community Review followed by 1 week of DCP Oversight Review. The intention of the "Community" review is to receive feedback not from other DCP team members, but rather the  scientific community.
+
+In practice, most RFCs are reviewed as a free-for-all during the community review period and then is followed by one week of radio silence during the "Oversight" review period since almost all feedback will have been resolved during the community review period.
+
+We propose removing Oversight review altogether. A two week minimum community review period still exists but the  "community" refers to all members of the DCP working on any of the chartered organizations instead of the scientific community only. This largely reflects the behavior of the review process today.
```

Comment:
To be precise, we don't have _chartered organizations_. We have _chartered projects_. 

_Each HCA DCP project must have a charter that specifies its scope, responsibilities, and leadership roles._ See the README - https://github.com/HumanCellAtlas/dcp-community/tree/master/charters